pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -125,3 +128,27 @@ func (r *Reconciler) reconcileIngress(ctx context.Context, dm *v1alpha1.DomainMa
 
 	return ingress, err
 }
+
+func (r *Reconciler) reconcileDomainClaim(ctx context.Context, dm *v1alpha1.DomainMapping) error {
+	dc, err := r.netclient.NetworkingV1alpha1().ClusterDomainClaims().Get(ctx, dm.Name, metav1.GetOptions{})
+	if apierrs.IsNotFound(err) {
+		if dc, err = r.netclient.NetworkingV1alpha1().ClusterDomainClaims().Create(ctx, &netv1alpha1.ClusterDomainClaim{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:            dm.Name,
+				OwnerReferences: []metav1.OwnerReference{*kmeta.NewControllerRef(dm)},
+			},
+		}, metav1.CreateOptions{}); err != nil {
```

Comment:
I'd move to `resources/` and use that in the table test, it should cut down on boilerplate.

Reply:
cool, good idea, willdo

Reply:
(done)